### ISSUE TITLE
event-stream `Cannot read property 'on' of undefined`

### ISSUE THREAD

**Issue by mightyiam** [verbose/verb#56]
Running `$ verb readme` in [this tree](https://github.com/PolicyStat/mde/tree/c708da8d422025eb2c6571784db0043e1977ff4f) results in:
![screenshot_2015-05-12_17-54-23](https://cloud.githubusercontent.com/assets/635591/7590289/6c455756-f8d0-11e4-90c8-b1093789f036.png)

**Comment by mightyiam** [comment]
Until now I've been fine with a single `.verb.md` file that generates a `README.md`.

In this project I'm going for a bigger documentation scheme, with multiple files.

So this is the first time that I write a `verbfile.js` and it failed.

Since this is the beginning of the project I'll workaround this by putting everything is the `.verb.md` for now, until this is resolved and then I'll split it up.

**Comment by jonschlinkert** [comment]
well that's not good. it should be seamless. I might know what it is. give me a few

**Comment by jonschlinkert** [comment]
can you share the contents of your verbfile?

**Comment by mightyiam** [comment]
See the tree linked in the original comment.

**Comment by jonschlinkert** [comment]
sorry I missed that, thanks!

**Comment by jonschlinkert** [comment]
is there a package.json in the actual project? i

**Comment by jonschlinkert** [comment]
actually I was assuming you put that up for the issue, it might be the project I just looked at the one file....

**Comment by jonschlinkert** [comment]
I'm pulling it down now

**Comment by mightyiam** [comment]
That tree is for the issue.

**Comment by mightyiam** [comment]
There is no `package.json` checked in. It is produced by some code. To produce the `package.json`, run `$ npm i auto-package && node package.js`.

**Comment by jonschlinkert** [comment]
what about `doc/src/README.md`?

**Comment by mightyiam** [comment]
That doesn't exist. When I tried creating one with just `foo\n` in it, I got the same error.

**Comment by jonschlinkert** [comment]
I'm not sure what you mean. where would the readme come from if there isn't a template or you didn't write one manually? sorry, I might have missed something

**Comment by mightyiam** [comment]
I didn't include the source `README.md`. I should have.

What I meant is that it doesn't seem to matter whether the source `README.md` exists or not. I get the same error.

**Comment by jonschlinkert** [comment]
k, try completely uninstalling both verb and verb-cli, `npm cache clean`, and then install the latest of both, and then please let me know if that works.

**Comment by mightyiam** [comment]
Seems like I'm still getting the same error.
![screenshot_2015-05-12_21-01-42](https://cloud.githubusercontent.com/assets/635591/7594555/1f19ebda-f8ea-11e4-9cdc-0e28dd575f06.png)
I've updated [the branch](https://github.com/PolicyStat/mde/tree/verbfile). Now it doesn't have the `.verb.md` and it does have the `doc/src/README.md` and I've also changed the name of the task to `default`, for comfort.

I cleaned the cache twice and made sure I don't have `verb-cli` or `verb` installed globally (I install them both locally).

I've made sure that I uninstalled them locally and that I then install the latests.
